### PR TITLE
1657#/cli_jvm_options

### PR DESCRIPTION
**Pull request by dvagnero**
#1657 
Docs updates for routerExtraJvmOpts and cliExtraJvmOpts
Needs backport to 2.2.x

### CODE REVIEW

File: documentation/asciidoc/topics/yaml/container_extra_jvm_opts.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 spec:
   container:
     extraJvmOpts: "-XX:NativeMemoryTracking=summary"
+    gossipRouterExtraJvmOpts: ""
```
Comment (dvagnero) [inline documentation/asciidoc/topics/yaml/container_extra_jvm_opts.yaml:4]: @ryanemerson Hi, could you please think about values for these two fields in this example? I'm looking for something more generic then disabling fips

Comment (ryanemerson) [reply documentation/asciidoc/topics/yaml/container_extra_jvm_opts.yaml:4]: The field name is `routerExtraJvmOpts` not `gossipRouterExtraJvmOpts`.

The current `extraJvmOpts` value is valid for both the `cliExtraJvmOpts` and `routerExtraJvmOpts` fields as it's a generic JVM arg, so we can keep it simple and set it for all three.

Users may want to tune the JVM used by these components, so generic JVM args such as `-Xms4` and `-Xmx` can be specified, however the only specific use-case example we have is FIPs.

Comment (dvagnero) [reply documentation/asciidoc/topics/yaml/container_extra_jvm_opts.yaml]: Thank you. I went with placeholders

**Comment by ryanemerson** [comment]
Thanks @dvagnero

**Pull request merged**